src/System.Net.Primitives/tests/FunctionalTests/LoggingTest.cs

```diff
@@ -25,10 +25,8 @@ public void EventSource_ExistsWithCorrectId()
         }
 
         [Fact]
-        [SkipOnTargetFramework(
-            ~TargetFrameworkMonikers.Netcoreapp, 
-            "NetFramework: NetEventSource is only part of .NET Core;" + 
-            "UapAot: RemoteExecutorConsoleApp.exe crashes trying to load netstandard.dll - StrongName issue")]
+        [ActiveIssue("https://github.com/dotnet/corefx/issues/19909", TargetFrameworkMonikers.Uap)]
```

Comment:
@AtsushiKan has a point in that SOTF attribute reasons are sometimes free text, sometimes ID's. Eg `[SkipOnTargetFramework(TargetFrameworkMonikers.Uap, '1234')]` When we next want to do a pass to find tests disabled against closed issues (which happens regularly) it's a little easier to filter out URLs from random text than numbers.

I don't really care either eway so long as we're consistent.